packages/core/ios/RNEmbrace/EmbraceManager.swift

```diff
@@ -22,6 +23,34 @@ private let EVENT_NAME_KEY = "name"
 private let EVENT_TIMESTAMP_KEY = "timeStampMs"
 private let EVENT_ATTRIBUTES_KEY = "attributes"
 
+class SDKConfig: NSObject {
+    public var appId = "";
+    public var appGroupId: String? = nil;
+    public var disableCrashReporter = false;
+    public var disableAutomaticViewCapture = false;
+    public var endpointBaseUrl: String? = nil;
+
+    public init(from: NSDictionary) {
+        for (key, value) in from {
+            if let key = key as? String {
+                // Keys defined in packages/core/interfaces/Config.ts
+                if key == "appId", let value = value as? String {
+                    appId = value
+                } else if key == "appGroupId", let value = value as? String {
+                    appGroupId = value
+                } else if key == "disableCrashReporter", let value = value as? Bool {
+                    disableCrashReporter = value
+                } else if key == "disableAutomaticViewCapture", let value = value as? Bool {
+                    disableAutomaticViewCapture = value
+                } else if key == "endpointBaseUrl", let value = value as? String {
+                    endpointBaseUrl = value
+                }
+            }
+        }
+    }
```

Comment:
I dont see any advantage to use for if we are going to check with if each variable inside. Maybe just decode the json with JSONDecoder (but it will fail if there is a typo error, you could handle the error)

or just something like this

```self.appId = from["appId"] as? String ?? ""
  self.appGroupId = from["appGroupId"] as? String
  self.disableCrashReporter = from["disableCrashReporter"] as? Bool ?? false
  self.disableAutomaticViewCapture = from["disableAutomaticViewCapture"] as? Bool ?? false
  self.endpointBaseUrl = from["endpointBaseUrl"] as? String```